app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render "shared/navbar" %>
+<h1 class=" text-2xl text-center font-extrabold">Recipe on Rails</h1>
+<div class="flex flex-wrap gap-2 justify-center  p-6">
+  <% @recipes.includes(:user).each do |r| %>
+      <%= render "shared/recipe_card", recipe: r %>
+  <% end %>
+</div>
```

Comment:
- [ ] Please make sure to display recipes ordered by the newest on the public recipes page as requested in the project requirements. Currently, recipes are not displayed ordered by the newest.

![image](https://github.com/sylvester-wamaya/Capstone-Project-Recipe/assets/83803414/16bab6dd-e6e4-4e89-a7e8-9361a94cd2a4)